pkg/utils/watchChaosContainer.go

```diff
@@ -49,9 +49,28 @@ func GetChaosContainerStatus(experimentDetails *ExperimentDetails, clients Clien
 			}
 		}
 
+	} else if pod.Status.Phase == corev1.PodPending {
```

Comment:
In case of podPending state, anything over a few seconds would mean the pod is not schedulable - very unlikely it will be scheduled after 180s, say for ex (this would be possible only if some other pod gets deleted, in case of pending due to resource constraint)

Reply:
However, we will be giving the benefit of doubt in case of delayed imagePulls on slower nodes - and hence go w/ the current timeout of 180s